grc/models.py

```diff
@@ -6,9 +6,11 @@
 from sqlalchemy_utils import StringEncryptedType
 from sqlalchemy_utils.types.encrypted.encrypted_type import AesEngine
 from grc.business_logic.data_structures.application_data import ApplicationData
+from grc.utils.logger import LogLevel, Logger
 
 db = SQLAlchemy()
 secret_key = os.environ.get('SQLALCHEMY_KEY', '')
+logger = Logger()
```

Comment:
I wouldn't add to the models either for similar reasons as my comment regarding the jobs